libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserTests.java

```diff
@@ -55,7 +55,7 @@ public void testPolicyBuilder() throws IOException {
             .parsePolicy();
         Policy expected = new Policy(
             "test-policy.yaml",
-            List.of(new Scope("entitlement-module-name", List.of(new FileEntitlement("test/path/to/file", "read_write"))))
+            List.of(new Scope("entitlement-module-name", List.of(FileEntitlement.create("test/path/to/file", "read_write"))))
```

Comment:
It's probably worth having explicit test(s) that check the behavior when eg a ctor and method are both annotated, similar to `ManyConstructorsEntitlement` already here. ie let's not rely on FileEntitlement, let's have explicit tests for this feature of the parser.

Reply:
I will add these.